### DISCUSSION TITLE
Improve Measure Team Effectiveness

### DISCUSSION THREAD

**Discussion by mschlupp** [microsoft/vsts-extension-retrospectives#284] [Ideas]
# Current state
The questions can be answered, however the results cannot be seen anywhere.
Given, that this feature works for a future release, I would like to share some ideas that would make the feature a lot more useful

## Track and visualize
In order to track the evolution of the team I would like to have access to the voting results for at least 6 months, better a year. It would be great to be able to visualize the results directly in Azure DevOps
E.g. similar to
![grafik](https://user-images.githubusercontent.com/11273097/140077720-6f138a00-826e-475b-820e-0f55605ac268.png)

## Customize the questions
Enable the user to ask custom question to the teams in order use the feature as a more general voting feature in the retrospectives plugin.

**Comment by evelienmoriau** [comment]
any visibility when the feature to see the Team´s result will be live? Further, it would be nice to inform to have an information pop-up that team results are not visibile yet.

**Comment by starkca** [comment]
I can see the Team's results by selecting the three dots and selecting "Show Retrospective Summary".  However, I would like two additional items:
1.  To be able to add one or two line items to be tracked at each retrospective.
2.  To be able to see results over time.

Are these two functions available now?  If so, then how can I see them?  If not, what is the ETA for these functions?

**Comment by johnstotko** [comment]
I think the teams result's shown in the "Show Retrospective Summary" is an average of the teams votes. However, I'd also like to also see a min and max for the questions as well. That way I'd be able to get an idea of the distribution of votes.

Currently, I wouldn't be able to tell if everyone gave 8's on a question or if half the team gave 10's and the other half gave 6's. The latter scenario would be an important thing to discuss during retro.

Also, the number of people who submitted the questionnaire would be helpful.

**Comment by DanTheManPearson** [comment]
Ditto the requests to be able to see results over time. Also, a way to query the results or use them in analytics views so they can be pulled into something like Power BI with other AzDO data.

**Comment by starkca** [comment]
Yay, thanks for agreeing with this suggestion.  Could be an easy and effective way to gauge Team Health over a period of time.  ☺


Sincerely,

Catherine


Catherine Stark
ITT/CN – Scrum Master for Cloud and Emerging Technologies
https://social.intra.corpintra.net/groups/cloud-and-emerging-technologies
[cet_logo_black_375x91]

**Comment by sandyteenan** [comment]
The ability to a) customize the questions to our own organization and b) see the trend of team effectiveness surveys over time would be hugely beneficial! Exposing the assessment summary via API would let us build our own reporting and visualization.

**Comment by dstoddar** [comment]
All, you can now see the Team Assessment results under retrospective summary and we are working on building the history into the history tab! Hopefully this will be released soon!
![image](https://user-images.githubusercontent.com/75645919/191650043-d15a8584-82e2-43ec-aff0-66507254c2c3.png)
![image](https://user-images.githubusercontent.com/75645919/191650136-54a202bb-b7fe-46e7-8719-0c3042ff5bea.png)
